src/Microsoft.AspNetCore.Server.IISIntegration.Tools/Program.cs

```diff
@@ -20,21 +20,23 @@ public static int Main(string[] args)
             app.HelpOption("-h|--help");
 
             var publishFolderOption = app.Option("--publish-folder|-p", "The path to the publish output folder", CommandOptionType.SingleValue);
+            var frameworkOption = app.Option("-f|--framework <FRAMEWORK>", "Target framework of application being published", CommandOptionType.SingleValue);
```

Comment:
We could. At the moment we don't say anything for --publish-folder so I did not add it here either. The reality is that typically people will not run it manually. Rather it will be in project.json in the post publish script section (created by our templates)